<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialDialog</class>
 <widget class="QDialog" name="SerialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>串口配置</string>
     </property>
     <widget class="QLabel" name="TEXT2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>66</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="TEXT4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>148</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>校验方式：</string>
      </property>
     </widget>
     <widget class="QLabel" name="TEXT1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>串口名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="TEXT5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>188</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>停止位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="TEXT3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>107</y>
        <width>70</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>数据位：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_stop">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_name">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>串口T1/R1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>串口T2/R2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>串口T3/R3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>串口T4/R4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>串口T5/R5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>串口T6/R6</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spin_data">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_pairty">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无校验</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>奇校验</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>偶校验</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="combo_baud">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>48400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
